src/material-experimental/mdc-slider/slider.ts

```diff
@@ -48,6 +55,173 @@ export interface MatSliderDragEvent {
   value: number;
 }
 
+/**
+ * The visual slider thumb.
+ *
+ * Handles the slider thumb ripple states (hover, focus, and active),
+ * and displaying the value tooltip on discrete sliders.
+ */
+@Component({
+  selector: 'mat-slider-visual-start-thumb, mat-slider-visual-end-thumb',
+  templateUrl: './slider-thumb.html',
+  host: {
+    'class': 'mdc-slider__thumb',
+    '(mouseenter)': '_onMouseEnter()',
+    '(mouseleave)': '_onMouseLeave()',
```

Comment:
These two listeners don't seem to be updating any data bindings so we could bind them outside of the `NgZone`.

Reply:
I tried this but I got a type error saying a class method is not a function. `_onMouseEnter()`. Is it ok to leave this as is?